### ISSUE TITLE
Add term osteoporosis of femoral neck

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10380]
**Preferred term label:**
Osteoporosis of femoral neck

**Synonyms**
Osteoporotic femoral neck

**Definition (free text, please give PubMed ID)**
Osteoporosis affecting the femoral neck
PMID: 24771492

**Parent term (use hpo.jax.org/app)**
Localized osteoporosis HP:0040161

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Osteoporosis (PMID: 26882774)
Femoral neck fracture

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000779
- label: Osteoporotic femoral neck
- parents: [Abnormal femoral neck morphology (HP:0003367)](https://hpo.jax.org/app/browse/term/HP:0003367)
- pmid: PMID:35378630
- def: Low bone mass localized in the neck of the femur. Osteoporosis is defined based on a T-score of -2.5 or lower (bone mineral density of 2.5 or more standard deviations below that of a normal young adult).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10380
- synonyms: Osteoporosis of femoral neck(NONE)
